apps/etterlatte-oppdater-behandling/src/main/kotlin/regulering/OmregningsHendelserBehandlingRiver.kt

```diff
@@ -36,8 +38,12 @@ internal class OmregningsHendelserBehandlingRiver(
         context: MessageContext,
     ) {
         logger.info("Mottatt omregningshendelse")
-
         val omregningData: OmregningData = packet.omregningData
+
+        if (omregningData.revurderingaarsak != Revurderingaarsak.REGULERING) {
```

Comment:
Er det meningen at alle Revurderingsaarsaker skal omregnes? Og ikke kun `Revurderingsaarsak.OMREGNING` ?

Reply:
I utgangspunktet nei. Omregninger skal være årsak OMREGNING eler REGULERING. Men så vil det være noen tilfeller hvor vi ønsker å bruke andre årsaker som inntektsjustering for eksempel. Men da må det være en eksplisitt flyt for det.

Reply:
Men kanskje vi ønsker å fortatt bruke årsak OMREGNING når det er en automatisk inntektsjustering. Men det får vi se nærmere på når vi jobber med det.